turte_graphics/turtle.js

```diff
@@ -0,0 +1,111 @@
+class Turtle {
+  constructor(x, y) {
+    this.x = x;
+    this.y = y;
+    this.position = [x, y];
```

Comment:
You don't really need both `this.position` *and* `this.x`/`this.y`. You only need one or the other. Otherwise you're creating duplication and have to make sure to keep them in sync, which can lead to problems down the road if you forget to update one but not the other. I think I would actually just keep `position` *but* make it into a Hash/Object. I.e. `this.position = {x: x, y: y}`. Although using a small array like this is not _terrible_, you have to remember that the first element is `x` and the second one is `y`, which adds cognitive burden, and you could accidentally reverse them. Also, `this.position.x` is more descriptive to the reader than `this.position[0]`.

Reply:
I think I would call them `currentPosition` and `currentDirection`

Reply:
I couldn't get it to work with the object, but I did remove the redundance of this.position

Reply:
https://github.com/rosadiaz/homeworks/pull/3

Reply:
You can dispose of `this.position` completely (if you're not going with the Object route)